File: py/class_simple.py
class Car:
    def __init__(self, brand, model):
        self.brand = brand
        self.model = model
    
    def start(self):
        return f"{self.brand} {self.model}'s engine started!"
    def stop(self):
        return f"{self.brand} {self.model}'s engine stoped!"

my_car = Car("TOGG", "T10X")
print(my_car.start())
print(my_car.stop())

File: dart/class_simple.dart
class Car {
  String brand;
  String model;

  Car({required this.brand, required String this.model});

  String start() {
    return "${this.brand} ${this.model}'s engine strated!";
  }
}

void main(List<dynamic> args) {
  Car car = Car(brand: "TOGG", model: "T10X");
  print(car.start());
}
